src/Task.gren

```diff
@@ -145,70 +150,38 @@ If it fails, the whole thing fails!
 
 -}
 map2 : (a -> b -> result) -> Task x a -> Task x b -> Task x result
-map2 func taskA taskB =
-    taskA
-        |> andThen
-            (\a ->
-                taskB
-                    |> andThen (\b -> succeed (func a b))
-            )
+map2 =
+    Gren.Kernel.Scheduler.map2
```

Comment:
`map2` is being used in the implementation of `sequence`, which means that `sequence` is now concurrent as well.

Could you find a new implementation for `sequence` that is non-concurrent?

Reply:
My bad, and well spotted! Have defined it using `andThen` now